### PR TITLE
Audit PR: Only add/remove labels when needed

### PR DESCRIPTION
**Pull request by gnarf**
Without this the bot gets super spammy in the issue log even when the state doesn't change...

screenshot 2015-02-07 15 59 47

### CODE REVIEW

File: lib/pr.js [hunk 1]
```diff
@@ -106,6 +106,8 @@ Audit.prototype._audit = function() {
 };
 
 Audit.prototype.applyLabels = function( labels, state ) {
+	var audit = this;
+	var pr = this.options.pr;
 	var addLabel, removeLabel;
```
Comment (scottgonzalez) [inline lib/pr.js]: We're still `onevar` for now.

File: lib/pr.js [hunk 2]
```diff
@@ -116,17 +118,31 @@ Audit.prototype.applyLabels = function( labels, state ) {
...
+			if (labelNames.indexOf(removeLabel) !== -1) {
+				audit.debug("removing label " + removeLabel + " from PR #" + pr);
+				ops.push(audit.repo.removeLabel({
+					pr: pr,
+					label: removeLabel
+				}));
+			}
+			audit.debug("doing " + ops.length + " label operations");
+			return Promise.all(ops);
+		});
```
Comment (scottgonzalez) [inline lib/pr.js]: Lots of spacing issues.

**Comment by scottgonzalez** [comment]
I filed #11 based on your earlier comment, so you can update the commit message too.

**Comment by gnarf** [comment]
Pushed style fixes, fell out of "jQuery Style Mode" so I'd appreciate another scrying eye

**Pull request closed without merging**